web/src/features/fba/components/infoPanel/InfoAccordion.tsx

```diff
@@ -20,24 +47,35 @@ const StyledAccordionSummary = styled(AccordionSummary)(() => ({
 }))
 
 // A component for rendering the provided title and content in an accordion format in the info panel.
-const InfoAccordion = ({ accordionDetailBackgroundColour, children, defaultExpanded, title }: InfoAccordionProps) => {
+const InfoAccordion = ({
+  accordionDetailBackgroundColour,
+  children,
+  defaultExpanded,
+  title,
+  advisoryStatus
+}: InfoAccordionProps) => {
   const theme = useTheme()
 
   return (
     <Accordion data-testid="info-accordion" disableGutters defaultExpanded={defaultExpanded} elevation={0}>
-      <StyledAccordionSummary expandIcon={<ExpandMoreIcon />}>
-        <Typography
-          data-testid="info-accordion-title"
-          sx={{
-            color: theme.palette.primary.main,
-            fontWeight: 'bold',
-            paddingLeft: '1.25rem'
-          }}
-          variant="h6"
-        >
-          {title}
-        </Typography>
-      </StyledAccordionSummary>
+      <Box>
+        <StyledAccordionSummary expandIcon={<ExpandMoreIcon />}>
+          <Typography
+            data-testid="info-accordion-title"
+            sx={{
+              color: theme.palette.primary.main,
+              fontWeight: 'bold',
+              paddingLeft: '1.25rem'
+            }}
+            variant="h6"
+          >
+            {title}
+          </Typography>
+        </StyledAccordionSummary>
+        {advisoryStatus && (
+          <AdvisoryStatusBar data-testid="advisory-status-bar" barColour={getAdvisoryBarColour(advisoryStatus)} />
```

Comment:
Nit: you could encapsulate `getAdvisoryBarColour` in `AdvisoryStatusBar` since it's only used by that component